assets/javascripts/discourse/services/doc-category-sidebar.js

```diff
@@ -31,6 +32,13 @@ export default class DocCategorySidebarService extends Service {
   }
 
   get activeCategory() {
+    if (
```

Comment:
Maybe instead of this we could check if the current route is an `admin` route or check if the sidebar state is currently using the admin panel, then return nil:

```javascript
this.sidebarState.currentPanel?.key === ADMIN_PANEL;
```

We don't show this docs sidebar in admin anywhere, I think this will save us from having to add more excluded entities in future.

Reply:
Your specs should still be able to remain the same